recipes/click-completion/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "click-completion" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "079fb138887d4de12a0b7fbebf8d92d396b7c1a9c49f63475d9f3909d2588976" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/d8/10/e2eead22c1893c2bd717b4dee55c72de5793fce38ee231211dd3858b012a/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
could this use pip https://conda-forge.org/docs/meta.html#use-pip

Reply:
Fixed